docs/antora.yml

```diff
@@ -10,6 +10,7 @@ asciidoc:
   attributes:
     # The full major.minor.patch version, which is used as a variable in the docs for things like download links
     full-version: '6.0.0-SNAPSHOT'
+    last-oss-version: '5.5.0'
```

Comment:
Just thinking again, is this the right name?
I think it should be `os-version` and `ee-version` to force you to think about what version you want to reference.

Reply:
Looks like this is only for OSS as we always release EE (so not needed)
May be `latest-oss-version`. This will also cover the case where we are in the process of releasing current MAJOR/MINOR
I think prefer its prepended with some test to make it very clear

Reply:
> Looks like this is only for OSS as we always release EE (so not needed)

Today, yes - but 3 months ago it was different.
If you aren't aware of the nuance it's easy to use `full-version` not realising it's really EE-only in some cases.

Reply:
I agree with Jack's naming suggestion - I was looking at this PR last week and there's something about the overall logic that was not quite right. If the versions are diverging we need a variable for each that we can use in the various places.